srcalc/src/test/java/gov/va/med/srcalc/vista/RpcVistaPatientDaoTest.java

```diff
@@ -157,4 +160,36 @@ public final void testLabResponseNoUnits()
         // Insure the lab was not added and no exceptions occurred.
         assertEquals(1, patient.getLabs().size());
     }
+    
+    @Test
+    public final void testHealthFactorsValid()
+    {
+        final VistaProcedureCaller caller = mockVistaProcedureCaller();
+        when(caller.doRpc(
```

Comment:
It would improve the other tests to put this mock return (or the empty one below) in `mockVistaProcedureCaller()` and then just override it as needed.